src/webapp/src/org/apache/solr/servlet/SolrDispatchFilter.java

```diff
@@ -359,6 +360,8 @@ protected void sendError(HttpServletResponse res, Throwable ex) throws IOExcepti
         code = 500;
       }
     }
+    // THIS IS YOUR PATCHED LINE HERE:
+    log.info( "minted extra logging: "+ (ex.getMessage() + trace) );
```

Comment:
and this isn't how log.info should be called.  Have you checked to see if this logs at debug?  if it doesn't have, you should still use the second argument to log the exceptions.  And exceptions that are fatal like this should log at warn or error.